README.md

```diff
@@ -123,16 +123,31 @@ Merging (needed to set `--signoff` and export environment variables):
 git duet-merge -v [any other git options]
 ```
 
+Rebasing (needed to set `--signoff` and export environment variables):
+
+``` bash
+git duet-rebase [any other git options]
+```
+
+Cherry-picking (needed to set `--signoff` and export environment variables):
+
+``` bash
+git duet-cherry-pick [any other git options]
+```
+
 Suggested aliases:
 
 ```
 dci = duet-commit
 drv = duet-revert
 dmg = duet-merge
+drb = duet-merge
```

Comment:
Yeah, I think the github UI is confused, I amended the commit and it's correct in my repo now:

https://github.com/thewoolleyman/git-duet/commit/master#diff-04c6e90faac2675aa89e2176d2eec7d8R145